collagraph/renderers/pyside/__init__.py

```diff
@@ -0,0 +1,104 @@
+from functools import lru_cache
```

Comment:
Make sure to use `functools.cache` or `functools.lru_cache(maxsize=None)` otherwise your cache will be bounded